inventory.json

```diff
@@ -1,63 +1,63 @@
-"arduino" : {
-	"name" : "Arduino Uno",
-	"total" : 5,
-	"working" : 5,
-	"allotted" : 3,
-	"available" : 2 
-},
-"raspberrypi" : {
-	"name" : "Raspberry Pi",
-	"total" : 4,
-	"working" : 4,
-	"allotted" : 2,
-	"available" : 2
-},
-"gtcrl" : {
-	"name" : "GTCRL-Conroller",
-	"total" : 6,
-	"working" : 6,
-	"allotted" : 3,
-	"available" : 3
-},
-"touchsensor" : {
-	"name" : "TTP223B Touch Sensor",
-	"total" : 5,
-	"working" : 3,
-	"allotted" : 2,
-	"available" : 1
-},
-"turbidity" : {
-	"name" : "Motion Sensor",
-	"total" : 4,
-	"working" : 4,
-	"allotted" : 3,
-	"available" : 1
-},
-"temperature" : {
-	"name" : "Infrared Temperature Sensor",
-	"total" : 6,
-	"working" : 5,
-	"allotted" : 2,
-	"available" : 3
-},
-"airquality" : {
-	"name" : "Air Quality Sensor",
-	"total" : 5,
-	"working" : 5,
-	"allotted" : 2,
-	"available" : 3
-},
-"jumper" : {
-	"name" : "Jumper Cables",
-	"total" : 9,
-	"working" : 9,
-	"allotted" : 5,
-	"available" : 4
-},
-"copperwire" : {
-	"name" : "Copper Wires",
-	"total" : 15,
-	"working" : 15,
-	"allotted" : 10,
-	"available" : 5
+"inventory" : {	
+	"microcontroller" : [
+		"arduino" : {
```

Comment:
See the red background? That tells you that the syntax is wrong. Run through a JSON validator before submitting